pkg/build/gobuild.go

```diff
@@ -969,6 +979,9 @@ func (g *gobuild) buildOne(ctx context.Context, refStr string, base v1.Image, pl
 			OSVersion:    cf.OSVersion,
 		}
 	}
+	if g.debug && !doesPlatformSupportDebugging(*platform) {
+		return nil, fmt.Errorf("debugging is not supported for %s", platform)
```

Comment:
I'd log and skip in this case, in case someone wants to build for all platforms but only intends to debug for certain platforms.

Reply:
Okay, gotcha. I tried to understand how "skip" would work, but it wasn't immediately obviously to me. Would we return a nil `oci.SignedImage` and a nil `error`?

Reply:
@imjasonh can you check on this?